libensemble/gen_classes/sampling.py

```diff
@@ -57,11 +56,11 @@ class UniformSampleDicts(Generator):
     mode by adjusting the allocation function.
     """
 
-    def __init__(self, _, persis_info, gen_specs, libE_info=None):
-        self.persis_info = persis_info
+    def __init__(self, _, persis_info, gen_specs, libE_info=None, **kwargs):
         self.gen_specs = gen_specs
-        self.libE_info = libE_info
+        self.persis_info = persis_info
         self._get_user_params(self.gen_specs["user"])
+        self.gen_specs["out"] = [("x", float, (self.n,))]
```

Comment:
Not sure about this. We are trying to represent an external gen here. Most external gens are not going to have this.

Reply:
Yeah, good point; I don't think `gen_specs["out"]` is necessary for this specific class